pfb-analysis/import/import_neighborhood.sh

```diff
@@ -41,6 +41,20 @@ NB_POSTGRESQL_PASSWORD - Default: gis
 "
 }
 
+# Function to import a shapefile (using 'dump' mode for quickness) and convert it to the target SRID
+function import_and_transform_shapefile() {
+    IMPORT_FILE="${1}"
+    IMPORT_TABLENAME="${2}"
+
+    echo "START: Importing ${IMPORT_TABLENAME}"
+    shp2pgsql -I -d -D -s 4326 "${IMPORT_FILE}" "${IMPORT_TABLENAME}" \
```

Comment:
Any specific reason to revert to static SRID 4326 here over using `-s "${NB_INPUT_SRID}"`? The cambridge neighborhood boundary is in a different projection.

I made the change back and it doesn't seem to have affected anything so far.